scripts/test/launchTestUtilities.ts

```diff
@@ -91,6 +91,7 @@ async function getVSCodeCliArgs(params: {
         extensionTestsEnv: {
             ['DEVELOPMENT_PATH']: projectRootDir,
             ['AWS_TOOLKIT_AUTOMATION']: params.suite,
+            ['TEST_DIR']: process.env.TEST_DIR,
             ...params.env,
```

Comment:
`...params.env` is already included so I guess the `['TEST_DIR']: process.env.TEST_DIR` assignement might not be necessary. But it's good for visibility/explicitness. 

Ideally, this would be passed as a parameter:

    ['TEST_DIR']: testDir,

but I don't know if the codepath is set up for that. Doesn't need to block this PR.